docs/en_US/Blog/HPOComparison.md

```diff
@@ -0,0 +1,323 @@
+# Hyperparameter Optimization Comparison
+*Posted by Anonymous Author*
+
+Comparison of Hyperparameter Optimization algorithms on several problems.
+
+Hyperparameter Optimization algorithms are list below:
+
+- Random Search
+- Grid Search
+- Evolution
+- Anneal
+- Metis
+- TPE
+- SMAC
+- HyperBand
+- BOHB
+
```

Comment:
Could you make a section to tell the detailed environment? Such as the configure of your machine, such as CPU core...

Reply:
OK.